perfil/election/management/commands/load_donations.py

```diff
@@ -0,0 +1,80 @@
+from textwrap import dedent
+
+from perfil.election.management.commands import election_keys
+from perfil.election.models import Donation, Election
+from perfil.utils.management.commands import ImportCsvCommand
+from perfil.utils.tools import parse_date, parse_decimal, parse_document
+
+
+class Command(ImportCsvCommand):
+
+    help = dedent("""
+        Import CSV generated by: https://github.com/rafapolo/tribuna
```

Comment:
Maybe we can start a proper documentation to add this kind of comments? What do you think?

Reply:
The only concern regarding this suggestion: I don't want to keep two documentation references. So if we document it elsewhere (let's say a `.md` file or GitHub wiki) we have to decide between:

1. assuming the risk of having non-matching documentation and command help text (that is to say, we might update the `.md` file and forget to change the `help` variable for the command)
1. or document everything elsewhere (`.md` or GitHub wiki in the exemple) creating a downside in terms of UX, that is to say: when the user is in fact at the terminal trying to import data, the help text wouldn't be helpful (would point him to the browser or to the `.md` file)

That's why IMHO documenting Django commands should be it in the help message of the command themselves: no duplicates to be kept in sync, and the information is delivered in the environment in which it is needed.

What do you think about it?

Reply:
Yeah, I am not sure if docs are the best approach. I don't have a define argument for defend it or not.